### PR TITLE
Added Get-Content and Set-Content support for items under FileShare

### PR DESCRIPTION
**Pull request by jianyunt**

### CODE REVIEW

File: AzurePSDriveStorageAccount.psm1 [hunk 1]
```diff
@@ -133,6 +133,12 @@ class Blobs : SHiPSDirectory
         }         
         return $obj     
     }
+
+    [object] SetContent([string]$content, [string]$path)
+    {
+        Write-Error -Message "Set-Content is not supported under 'Blobs' service. Try it again under Files\<Share> directory."
```
Comment (raghushantha) [inline AzurePSDriveStorageAccount.psm1]: 'Blobs' resource

File: AzurePSDriveStorageAccount.psm1 [hunk 2]
```diff
@@ -179,6 +185,12 @@ class Files : SHiPSDirectory
              [FileShare]::new($_.Name, $this.data.Context, "$fileshare$($_.Name);$($accountInfo)");
             }
     }
+
+    [object] SetContent([string]$content, [string]$path)
+    {
+        Write-Error -Message "Set-Content is not supported under 'Files' service. Try it again under its subfolders."
```
Comment (raghushantha) [inline AzurePSDriveStorageAccount.psm1]: 'Files' resource

File: AzurePSDriveStorageAccount.psm1 [hunk 3]
```diff
@@ -197,6 +209,12 @@ class Tables : SHiPSDirectory
     {     
         return @($this.result | Sort-Object Name)    
     }
+
+    [object] SetContent([string]$content, [string]$path)
+    {
+        Write-Error -Message "Set-Content is not supported under 'Tables' service. Try it again under Files\<Share> directory."
```
Comment (raghushantha) [inline AzurePSDriveStorageAccount.psm1]: 'Tables' resource

File: AzurePSDriveStorageAccount.psm1 [hunk 4]
```diff
@@ -215,6 +233,12 @@ class Queues : SHiPSDirectory
     {     
         return @($this.result | Sort-Object Name)  
     }
+
+    [object] SetContent([string]$content, [string]$path)
+    {
+        Write-Error -Message "Set-Content is not supported under 'Queues' service. Try it again under Files\<Share> directory."
```
Comment (raghushantha) [inline AzurePSDriveStorageAccount.psm1]: 'Queues' resource

File: AzurePSDriveStorageAccount.psm1 [hunk 5]
```diff
@@ -298,3 +419,42 @@ class Blob : SHiPSDirectory
...
+
+    # Save the text content to a local temp file because the Set-AzureStorageFileContent cmdlet takes file only
+    $tmpfile = [System.IO.Path]::GetTempFileName()
+
+    try {
+
+        Microsoft.PowerShell.Management\Set-Content -Path $tmpfile -Value $Content
+
+        # $content is the 'Value' passed in from Set-Content
+        # $path is the full path. e.g., Azure:\AutomationTeam\StorageAccounts\myaccount\Files\HelloWorld\hello.ps1".
```
Comment (raghushantha) [inline AzurePSDriveStorageAccount.psm1]: Remove references to internal subscription or resource names

Comment (jianyunt) [reply AzurePSDriveStorageAccount.psm1]: Updated.

File: AzurePSDriveStorageAccount.psm1 (hunk 4 above)
Comment (jianyunt) [reply AzurePSDriveStorageAccount.psm1]: Removed word service.

File: AzurePSDriveStorageAccount.psm1 (hunk 3 above)
Comment (jianyunt) [reply AzurePSDriveStorageAccount.psm1]: Removed word service.

File: AzurePSDriveStorageAccount.psm1 (hunk 1 above)
Comment (jianyunt) [reply AzurePSDriveStorageAccount.psm1]: Removed word service.

**Pull request merged**